### DISCUSSION TITLE
隐藏推荐提示

### DISCUSSION THREAD

**Discussion by github-lxb** [alovajs/alova#149] [General]
项目是使用的基于 vue3 的 uni-app，按照官方文档中的【隐藏推荐提示】，在 .env.development 中设置 VITE_ALOVA_TIPS=0 不生效，翻看源码，发现并没有这个条件判断是否打印提示

![image](https://github.com/alovajs/alova/assets/51040135/c9dbc2a6-d7f3-45d1-9d56-a4c18e41408a)

**Comment by JOU-amjs** [comment]
请查看这个[issues](https://github.com/alovajs/alova/issues/135)
